File: lending/src/main/java/com/lending/controller/HomeController.java
package com.lending.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;

@Controller
public class HomeController {
    @RequestMapping(value="/")
    public String index(){
        return "index";
    }
    @RequestMapping(value="/contact-us")
    public String contact(){
        return "contact-us";
    }
    @RequestMapping(value="/login")
    public String login(){
        return "login";
    }
}

File: !OtherAppVersions/borrow-me/borrow-me/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Linq;
using System.Threading.Tasks;
using Microsoft.AspNetCore.Mvc;
using borrow_me.Models;

namespace borrow_me.Controllers
{
    public class HomeController : Controller
    {
        public IActionResult Index()
        {
            return View();
        }

        [Route("kontakt", Name ="contact")]
        public IActionResult Contact()
        {
            return View();
        }

        [Route("logowanie", Name ="login")]
        public IActionResult Login()
        {
            return View();
        }

        [ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]
        public IActionResult Error()
        {
            return View(new ErrorViewModel { RequestId = Activity.Current?.Id ?? HttpContext.TraceIdentifier });
        }
    }
}
